### DISCUSSION TITLE
React Native Linear Gradient New Architecture adoption

### DISCUSSION THREAD

**Discussion by chakrihacker** [reactwg/react-native-new-architecture#42] [Libraries]
Hi 👋 

I am trying to migrate [React Native Linear Gradient](https://github.com/react-native-linear-gradient/react-native-linear-gradient) to Fabric Component

**Comment by sammy-SC** [comment]
Hello @chakrihacker, 

thank you for the effort! how is it going so far?

**Reply by chakrihacker** [reply]
Hey Sammy thanks for checking in will update you tomorrow

**Reply by chakrihacker** [reply]
Currently I am unable to generate codegen schema on Android. Will dig more

`react-native-fabric-linear-gradient:generateCodegenSchemaFromJavaScript`

**Comment by chakrihacker** [comment]
Hey @sammy-SC here's the initial repo https://github.com/FyndX/react-native-fabric-linear-gradient

Currently it just has stubs for linear gradient which doesn't do anything.

I am currently not able to build Android

Here are some details

```
Task :app:configureNdkBuildDebug[arm64-v8a] FAILED

C/C++: fcntl(): Bad file descriptor
C/C++: /Users/subramanyachakravarthy/Library/Android/sdk/ndk/21.4.7075529/build/core/build-binary.mk:651: Android NDK: Module example_appmodules depends on undefined modules: react_codegen_lineargradientview    
C/C++: /Users/subramanyachakravarthy/Library/Android/sdk/ndk/21.4.7075529/build/core/build-binary.mk:664: *** Android NDK: Note that old versions of ndk-build silently ignored this error case. If your project worked on those versions, the missing libraries were not needed and you can remove those dependencies from the module to fix your build. Alternatively, set APP_ALLOW_MISSING_DEPS=true to allow missing dependencies
```


Any inputs on what I am doing wrong would be helpful

**Comment by chakrihacker** [comment]
Hey, small update iOS is working fine 🎉. 
I am facing some issues with building an Android example app that supports the fabric.

**configureNdkBuildDebug** is failing on my machine

It might be related to these conversations

1. https://github.com/facebook/react-native/issues/33716
2. https://github.com/reactwg/react-native-releases/discussions/13#discussioncomment-2370055

**Comment by chakrihacker** [comment]
Hello RN Community I have an update for Linear Gradient on Fabric

Here's the initial version 

Github: https://github.com/FyndX/react-native-gradient
npm: https://www.npmjs.com/package/@fyndx/react-native-gradient

I have plans to  improve it and add Radial Gradient, and web support in future

Please provide your feedback
